login/css/styles.css

```diff
@@ -0,0 +1,946 @@
+
```

Comment:
In this .css file you have made a lot of spaces which resulted with huge unused lines, reduce it

Reply:
thank you. i m in the proccess)